.tekton/tasks/verify-smoke-e2e.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: stage-verify-e2e-test
+spec:
+  description: >-
+    This task is for running stage e2e test on RHTAP staging cluster
+  params:
+  - name: TASK_RETRIES
+    description: number of times the task will be rerun
+    default: "0"
+  workspaces:
+    - name: stage-e2e-git
+  steps:
+    - name: run-stage-verify-e2e-test
```

Comment:
Maybe this task should have a param so that it can be used to for staging and production?

Reply:
done